docs/user-guide/api-mediation/configuration-at-tls.md

```diff
@@ -1,23 +1,183 @@
+
+:::note
+Support for AT-TLS was introduced back in Zowe 1.24, however there was an issue that prevented startup in some versions. It is recommended to upgrade to 2.13 or newer for full support.
+:::
+
+To enable the AT-TLS profile and disable the TLS application in API ML, update `zowe.yaml` with following values under the respective component in the `zowe.components` section.
+
+```yaml
+components.*.spring.profiles.active: attls
+components.*.server.ssl.enabled: false
+components.*.server.internal.ssl.enabled: false
 ```
-components.*.certificate.keystore.file=<SAF-key-ring-from-AT-TLS-rule>
-components.*.certificate.keystore.type=JCERACFKS
-components.*.certificate.keystore.password=<keyring-password>
-components.*.certificate.keystore.alias=<certificate-alias-from-AT-TLS-rule>
+
+While API ML will not handle TLS on its own with AT-TLS enabled, the Mediation Layer needs information about the server certificate that is defined in the AT-TLS rule.
+
+Update the `zowe.yaml` file for each respective APIML component in the `zowe.components` sections with the path to the SAF Key ring from the AT-TLS rule and specify the alias that is used for Inbound communication:
+
+```yaml
+components.*.certificate.keystore.file: <SAF-key-ring-from-AT-TLS-rule>
+components.*.certificate.keystore.type: JCERACFKS
+components.*.certificate.keystore.password: password
+components.*.certificate.keystore.alias: <certificate alias / label from AT-TLS rule>
 ```
+
+Finally, if there is an outbound AT-TLS rule configured for the link between the API Gateway and z/OSMF, update or set the `zowe.zOSMF.scheme` to `http`.
+
 :::note
-This procedure does not configure AT-TLS on z/OS, but rather enables API ML to work with AT-TLS in place.
+AT-TLS is not yet supported in the API Cloud Gateway Mediation Layer component.
 :::
+
+## Security considerations
+
+Configuring AT-TLS for the Zowe API Mediation Layer needs careful consideration regarding the security settings, especially around the Client Certificate authentication feature in the Zowe API Mediation Layer components as well as for the Onboarded services that support the x.509 client certificates authentication scheme.
+
+In general terms, the outbound AT-TLS rules (i.e. to make a transparent https call through http) that are configured to send the server certificate should be limited to the services that __require__ service to service authentication, such as the case of the API Gateway authenticating with the Discovery Service.
+
+**Note:** The Discovery Service endpoints are not reachable by standard API Gateway routing by default.
+
+## Inbound and Outbout AT-TLS rules
+
+This section describes the suggested AT-TLS settings. It is meant as a guideline and helps showcase important aspects.
+
+### Inbound rules
+
+A generic inbound rule can be set for all of the core API Mediation Layer services:
+
+```pagent
+TTLSRule ApimlServerRule
+{
+  LocalAddr All
+  RemoteAddr All
+  LocalPortRange 10310-10320
+  Jobname ZWE*
```

Comment:
It could be customized, just add a comment about. It would be good to explain that ports and jobname are using as and - both must match.